UI/window-basic-main-profiles.cpp

```diff
@@ -333,8 +333,9 @@ void OBSBasic::RefreshProfileCache()
 			candidateName = entry.path().filename().u8string();
 		}
 
+		auto candidatePath = std::filesystem::u8path(profileCandidate);
 		foundProfiles.try_emplace(candidateName, OBSProfile{candidateName, entry.path().filename().u8string(),
-								    entry.path(), profileCandidate});
+								    entry.path(), candidatePath});
```

Comment:
This change is good in two ways - it fixes the crash and also avoids relying on implicit type conversion by the `OBSProfile` constructor.